src/JuliaWorkspaces.jl

```diff
@@ -4,6 +4,7 @@ import UUIDs, JuliaSyntax, TestItemDetection
 using UUIDs: UUID, uuid4
 using JuliaSyntax: SyntaxNode
 using Salsa
+using Documenter, DocStringExtensions
```

Comment:
We should definitely not have `Documenter` as a regular package dependency and load it here.

I don't know what `DocStringExtensions` is, but do we need that? Ideally we would just delete this entire line here.

Reply:
Well, without [DocStringExtensions.jl](https://docstringextensions.juliadocs.org/latest/) you cannot use this notation:
```
"""
    struct TestItemDetail
Details of a test item.
$(TYPEDFIELDS)
"""
```
I find this notation very useful because you do not have to repeat all the fields of a struct when documenting a struct. Which is also error-prone, because without using the macro `$TYPEDFIELDS`, if you add a field to the struct you can easily forget to add it to the documentation string.

There is no need to have `Documenter` as dependency of the package, but if we want to be able to use macros like `$TYPEDFIELDS` we need `DocStringExtensions` as dependency. It is very lightweight. 

But it is up to you if you want it or not. If you do not want it I can remove it, but it makes documenting structs more tedious and error prone.

Reply:
Ok, I see the value, but adding dependencies to the LS is painful because of the way we distribute it in the VS Code extension (we add all deps as git submodules). So in this case I'd prefer that we don't use it, just to not make the deployment via VS Code even more involved.

Reply:
I removed DocStringExtensions. But it means, if you add/remove a field to/from a struct in the future, please also update the doc string.